docker/rholang-cli/run-rhoscala

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -e
+
+ulimit -s unlimited
+
+java -jar ${RHOLANG_JAR} /tmp/input.rho
```

Comment:
```
Line 7:
java -jar ${RHOLANG_JAR} /tmp/input.rho
          ^-- SC2086: Double quote to prevent globbing and word splitting.
```

Reply:
If you quote it then `RHOLANG_JAR=/usr/lib/java/*` doesn't work the way you'd expect. Though, in this case I could be convinced.

Reply:
true, if you expect globbing, it's correct as written.